### ISSUE TITLE
Fails to identify device from user_agent string

### ISSUE THREAD

**Issue by werleo** [podigee/device_detector#17]
Device_name returns nil for user_agent string 
```'Mozilla/5.0 (Linux; Android 5.1.1; ONE E1003 Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.95 Mobile Safari/537.36'```

but returns proper OS info:
```
> c = DeviceDetector.new 'Mozilla/5.0 (Linux; Android 5.1.1; ONE E1003 Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.95 Mobile Safari/537.36'
> c.device_name
=> nil
> c.device_brand
=> nil
> c.os_name
=> "Android"
> c.os_full_version
=> "5.1.1"
```

**Comment by peteygao** [comment]
Device name is implemented as separate regexes for each device family, so we clearly don't have a regex for this device family. Is the Device's name `ONE E1003`?

**Comment by benzimmer** [comment]
This is fixed with the regex update in 0.9.

```ruby
> c = DeviceDetector.new 'Mozilla/5.0 (Linux; Android 5.1.1; ONE E1003 Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.95 Mobile Safari/537.36'
> c.device_name
=> "X"
> c.device_brand
=> "OnePlus"
> c.os_name
=> "Android"
> c.os_full_version
=> "5.1.1"
```

**Comment by werleo** [comment]
thank you.
